docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -5,9 +5,9 @@
 {kib} alerts can be created in a variety of apps including <<xpack-apm,*APM*>>, <<xpack-ml,*{ml-app}*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, <<uptime-app,*Uptime*>> and from <<management,*Management*>> UI. While alerting details may differ from app to app, they share a common interface for defining and configuring alerts that this section describes in more detail.
 
 [float]
-=== Alert flyout
+=== Create an alert
```

Comment:
All alerts share the following four properties in common: >

All alerts share the following four properties.